modules/apps/auxano-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+
+<div class="container-fluid" id="wrapper">
+	<header id="banner" role="banner">
+
+
+
+		<!-- Landing page description for company -->
+		<h2 class="landing-description">growing and increasing as a <strong>family</strong> <br> one cup at a time.</h2>
+		<div id="heading">
+			<h1 class="site-title">
+			
+				<#if show_site_name>
+					<span class="site-name" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+						Auxano
+					</span>
+				</#if>
+			</h1>
+		</div>
+
+		<#if !is_signed_in>
+			<a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+		</#if>
+
+		<#if has_navigation && is_setup_complete>
+			<#include "${full_templates_path}/navigation.ftl" />
+		</#if>
+	</header>
+
+	<section id="content">
+
+
+
+		<!-- New Arrivals Content -->
+		<div class="new-arrivals">
+			<div class="new-arrivals-content">
+				<p id="colombia">Colombia</p>
+				<h4 id="colombia-region">Tres Santos Colombia</h4>
+				<hr class="new-arrivals-break">
+				<a href="#"><h3 class="new-arrivals-link">New Arrivals</h3></a>
+			</div>
+		</div>
+
+
+
+
+		<!-- Coffee Tutorial Cards -->
+		<div class="card-container">
+
+			<div class="card">
+				<div id="image-learn" class="card-image">
+					
+				</div>
```

Comment:
Since there is no content in this ```<div />```, we can put the closing tag right next to the opening tag.

Ex.
```
<div id="image-learn" class="card-image"></div>
```